recipe/meta.yaml

```
@@ -18,8 +18,8 @@ requirements:
     - pkg-config  # [win]
   host:
     - python
-    - cython >=3.0.10
-    - libxml2 {{ libxml2 }}
+    - cython >=3.0.11
+    - libxml2 2.13.5
```

```suggestion
    - libxml2 {{ libxml2 }}
```
This would be sufficient, as our cbc points to 2.13.